### PR TITLE
(auxtel-hcu02.cp) OS Redeployment

### PR DESCRIPTION
**Pull request by dtapiacl**
This pull request updates the network configuration and tests for Centos 7 redeployment of auxtel-hcu02.cp.lsst.org.

### CODE REVIEW

File: spec/hosts/nodes/auxtel-hcu02.cp.lsst.org_spec.rb [hunk 1]
```diff
@@ -28,37 +28,41 @@
       it { is_expected.to compile.with_all_deps }
 
       include_examples 'baremetal no bmc'
-      include_context 'with nm interface'
-      it { is_expected.to have_nm__connection_resource_count(4) }
```
Comment (jhoblitt) [inline spec/hosts/nodes/auxtel-hcu02.cp.lsst.org_spec.rb:32]: Having a resource count for `network__interface` is a good idea to make sure extra interfaces aren't being created by accident.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
A resource count check would be a good ideal. Otherwise, this looks good.

File: hieradata/node/auxtel-hcu02.cp.lsst.org.yaml [hunk 2]
```diff
@@ -1,67 +1,21 @@
...
-      [ethernet]
-
-      [ipv4]
-      method=manual
-      address1=192.168.1.1/24
-
-      [ipv6]
-      method=disabled
-
-ccs_hcu::imanager: true
```
Comment (glennmorris) [inline hieradata/node/auxtel-hcu02.cp.lsst.org.yaml:67]: Why are you removing the ccs_hcu::imanager setting?
I would like it to stay.

Comment (jhoblitt) [reply hieradata/node/auxtel-hcu02.cp.lsst.org.yaml:67]: I missed that it was being removed.  Is there any reason we can't move this to the role?

Comment (glennmorris) [reply hieradata/node/auxtel-hcu02.cp.lsst.org.yaml:67]: Long term it should be in the role, yes.
But I don't want to affect hcu01 at the moment, where it seems imanager was installed by hand ages ago.

**Review by glennmorris — APPROVED** [review]
Looks good to me now, thanks.

**Review by cbarria — APPROVED** [review]
Looks Great 👍🏻

**Review by jhoblitt — APPROVED** [review]
LGTM

**Pull request merged**